### ISSUE TITLE
Pods NOT compatible with Siteground's SG Optimizer Plugin

### ISSUE THREAD

**Issue by ryruss** [pods-framework/pods#5769]
**Describe the bug**
When Pods is activated, the SG Optimizer will not cache the website. Link: https://wordpress.org/support/topic/it-doenst-work-if-you-have-installed-pods/#post-13099938

**To Reproduce**
1. Go to 'SG Optimizer plugin'
2. Click on 'Test'
3. See error

**Expected behavior**
Must see 'The URL is cached'

**Screenshots**
<img width="1277" alt="Screenshot 2020-07-09 at 8 22 51 PM" src="https://user-images.githubusercontent.com/8025499/87039703-25f96780-c222-11ea-99e3-143f801a7c1a.png">

## Pods Version
Version 2.7.21 

## WordPress Environment

PHP 7.0

</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Additional context**

**Possible Workaround**
https://wordpress.org/support/topic/it-doenst-work-if-you-have-installed-pods/#post-13099938

**Comment by ryruss** [comment]
All good after adding define('PODS_SESSION_AUTO_START', false); inside wp-config.php

Thank you for the support!

**Comment by sc0ttkclark** [comment]
Fixed via #5626
